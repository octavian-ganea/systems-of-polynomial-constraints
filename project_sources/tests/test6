x^2 + y^2 + z^2 - 2*x*y - 4* z + 4 = 0
z - 2 > 0.1

% (x-y)^2 + (z-2)^2 = 0
% We should obtain no solution (null) here because first equation is (x-y)^2 + (z - 2)^2 = 0
% Try commenting the second line and it should give x = y and z = 2.