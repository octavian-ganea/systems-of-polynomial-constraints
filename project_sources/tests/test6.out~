Parsing file test6
Parsing succeded. Results:
List of all variables:
Vector(x, y, z)
List of all polynomials:
Vector((1.0 y^2 + (4.0  + -4.0 z + 1.0 z^2)) + (-2.0 y)x + 1.0 x^2, -2.0  + 1.0 z)

List of desired signs : Vector(=, >)
Analysed polynomials: 
(1.0 y^2 + (4.0  + -4.0 z + 1.0 z^2)) + (-2.0 y)x + 1.0 x^2
-2.0  + 1.0 z
vars: Vector(x, y, z)

Univariate projection (on real axis) contains these polynomials: 
4.0  + -4.0 z + 1.0 z^2
-2.0  + 1.0 z
16.0  + -16.0 z + 4.0 z^2
256.0  + -512.0 z + 384.0 z^2 + -128.0 z^3 + 16.0 z^4
16.0  + -32.0 z + 24.0 z^2 + -8.0 z^3 + 1.0 z^4

There are 33 cells :
Map(x -> 0.0, y -> -1.0, z -> 0.9998627903884092)
Map(x -> 0.0, y -> 0.0, z -> 0.9998627903884092)
Map(x -> 0.0, y -> 1.0, z -> 0.9998627903884092)
Map(x -> 0.0, y -> -1.0, z -> 1.9998627903884092)
Map(x -> 0.0, y -> 0.0, z -> 1.9998627903884092)
Map(x -> 0.0, y -> 1.0, z -> 1.9998627903884092)
Map(x -> 0.0, y -> -1.0, z -> 1.9998694298767907)
Map(x -> 0.0, y -> 0.0, z -> 1.9998694298767907)
Map(x -> 0.0, y -> 1.0, z -> 1.9998694298767907)
Map(x -> 0.0, y -> -1.0, z -> 1.9998760693651723)
Map(x -> 0.0, y -> 0.0, z -> 1.9998760693651723)
Map(x -> 0.0, y -> 1.0, z -> 1.9998760693651723)
Map(x -> 0.0, y -> -1.0, z -> 1.9999380346825861)
Map(x -> 0.0, y -> 0.0, z -> 1.9999380346825861)
Map(x -> 0.0, y -> 1.0, z -> 1.9999380346825861)
Map(x -> -2.000002001152443, y -> -1.0000010008260192, z -> 2.0)
Map(x -> -1.0000020011524429, y -> -1.0000010008260192, z -> 2.0)
Map(x -> -2.0011524428653615E-6, y -> -1.0000010008260192, z -> 2.0)
Map(x -> -1.0000020017193403, y -> -1.0008260191095398E-6, z -> 2.0)
Map(x -> -2.0017193403571227E-6, y -> -1.0008260191095398E-6, z -> 2.0)
Map(x -> 0.9999979982806596, y -> -1.0008260191095398E-6, z -> 2.0)
Map(x -> -1.0000015008217107, y -> -5.004130095547699E-7, z -> 2.0)
Map(x -> -1.5008217105936755E-6, y -> -5.004130095547699E-7, z -> 2.0)
Map(x -> 0.9999984991782894, y -> -5.004130095547699E-7, z -> 2.0)
Map(x -> -1.0000010008260192, y -> 0.0, z -> 2.0)
Map(x -> -1.0008260191095398E-6, y -> 0.0, z -> 2.0)
Map(x -> 0.9999989991739809, y -> 0.0, z -> 2.0)
Map(x -> 0.0, y -> 1.0, z -> 2.0)
Map(x -> 1.0, y -> 1.0, z -> 2.0)
Map(x -> 2.0, y -> 1.0, z -> 2.0)
Map(x -> 0.0, y -> -1.0, z -> 3.0)
Map(x -> 0.0, y -> 0.0, z -> 3.0)
Map(x -> 0.0, y -> 1.0, z -> 3.0)
====== FINAL RESULT FOR test6 ================================
null
====================================================

