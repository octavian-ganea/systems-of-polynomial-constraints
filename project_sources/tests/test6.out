====================================================
Parsing file test6
Parsing succeded. Results:
List of all variables:
Vector(x, y, z)
List of all polynomials:
Vector((1.0 y^2 + (4.0  + -4.0 z + 1.0 z^2)) + (-2.0 y)x + 1.0 x^2, -2.0  + 1.0 z)

List of desired signs : Vector(=, >)
Analysed polynomials: 
(1.0 y^2 + (4.0  + -4.0 z + 1.0 z^2)) + (-2.0 y)x + 1.0 x^2
-2.0  + 1.0 z
vars: Vector(x, y, z)

Univariate projection (on real axis) contains these polynomials: 
4.0  + -4.0 z + 1.0 z^2
-2.0  + 1.0 z
16.0  + -16.0 z + 4.0 z^2
256.0  + -512.0 z + 384.0 z^2 + -128.0 z^3 + 16.0 z^4
16.0  + -32.0 z + 24.0 z^2 + -8.0 z^3 + 1.0 z^4

There are 15 cells
Map(x -> 0.0, y -> -1.0, z -> 1.0)
Map(x -> 0.0, y -> 0.0, z -> 1.0)
Map(x -> 0.0, y -> 1.0, z -> 1.0)
Map(x -> -2.0000010001062365, y -> -1.0, z -> 2.0)
Map(x -> -1.0000010001062365, y -> -1.0, z -> 2.0)
Map(x -> -1.000106236492826E-6, y -> -1.0, z -> 2.0)
Map(x -> -1.0000010003753592, y -> 0.0, z -> 2.0)
Map(x -> -1.0003753592931224E-6, y -> 0.0, z -> 2.0)
Map(x -> 0.9999989996246407, y -> 0.0, z -> 2.0)
Map(x -> -9.99973676307775E-7, y -> 1.0, z -> 2.0)
Map(x -> 0.9999990000263237, y -> 1.0, z -> 2.0)
Map(x -> 1.9999990000263237, y -> 1.0, z -> 2.0)
Map(x -> 0.0, y -> -1.0, z -> 3.0)
Map(x -> 0.0, y -> 0.0, z -> 3.0)
Map(x -> 0.0, y -> 1.0, z -> 3.0)
====== FINAL RESULT FOR test6 ================================
null

